src/osemgrep/reporting/Status_report.ml

```diff
@@ -67,12 +67,35 @@ let pp_status ~num_rules ~num_targets ~respect_git_ignore lang_jobs ppf =
           "<multilang>"
       | Xlang.L (l, _) -> Lang.to_lowercase_alnum l
     in
+    (* TODO: Improve Performance? *)
+    let lang_jobs =
+      lang_jobs
+      (* Unpack each job and transform lang to xlang *)
+      |> Common.map (fun Lang_job.{ xlang; targets; rules } ->
+             (xlang_label xlang, rules, targets))
+      (* Account for duplicate targets for multilang rules *)
```

Comment:
The question then is why do we have those different lang_jobs with the same xlang?
Instead of fixing the report, we should fix the issue at the root, that is when we build those lang_jobs.

Reply:
Maybe there is a good reason for having the same xlang with different targets, but then we need to add back the reason in comment here.

Reply:
Added a comment explaining the situation!